meson_options.txt

```diff
@@ -58,6 +58,7 @@ option('caca', type: 'feature', value: 'auto', description: 'CACA')
 option('cocoa', type: 'feature', value: 'auto', description: 'Cocoa')
 option('d3d11', type: 'feature', value: 'auto', description: 'Direct3D 11 video output')
 option('direct3d', type: 'feature', value: 'auto', description: 'Direct3D support')
+option('dmabuf-wayland', type: 'feature', value: 'auto', description: 'VAAPI or DRM (Wayland support)')
```

Comment:
We don't need this actually since it's not a build option. The vo is enabled if you have the correct combination of features.